bin/cheritest/cheritest.c

```diff
@@ -1638,15 +1638,11 @@ static const struct cheri_test cheri_tests[] = {
 	{ .ct_name = "test_unaligned_capability_copy_memcpy",
 	  .ct_desc = "Check that a memcpy() of valid capabilities to an "
 		     "unaligned destination fails",
```

Comment:
Please update the description for these two to match.

Reply:
Better?